src/components/application_manager/src/message_helper.cc

```diff
@@ -1602,6 +1602,8 @@ void MessageHelper::SendPolicyUpdate(
     object[strings::msg_params][hmi_request::retry][i] = retries[i];
   }
   ApplicationManagerImpl::instance()->ManageHMICommand(message);
+  // to Reset the update_required_ flag
+  policy::PolicyHandler::instance()->OnUpdateRequestSentToMobile();
```

Comment:
@madhuy I see that `OnUpdateRequestSentToMobile()` method performs a lot of logic but not only `update_required` flag resetting. Are you sure you need to start this logic here?